config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  
+  resources :items, only: [:create, :destroy]
```

Comment:
I would typically next my Items under my User like this: 

```
  resources :users, only: [:show] do
    resources :items, only: [:create, :destroy]
  end
```

your call if that makes sense with your workflow.